blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/PipelineFolderImpl.java

```diff
@@ -97,7 +97,7 @@ public BlueFavorite favorite(@JsonBody BlueFavoriteAction favoriteAction) {
 
     @Override
     public Link getLink() {
-        return OrganizationImpl.INSTANCE.getLink().rel("pipelines").rel(PipelineImpl.getRecursivePathFromFullName(this));
+        return OrganizationImpl.INSTANCE.getLink().rel("pipelines").rel(AbstractPipelineImpl.getRecursivePathFromFullName(this));
```

Comment:
Maybe, I find it hard too, we have precedence on doing it with AbstractRunImpl as well. I have seen such usage elsewhere, such names are really opinion oriented, no hard and fast. For the same reason, you could have an 'abstract' class without Abstract name in it:)